pkg/cmd/attestation/artifact/digest/digest.go

```diff
@@ -0,0 +1,53 @@
+package digest
+
+import (
+	"crypto/sha256"
+	"crypto/sha512"
+	"encoding/hex"
+	"fmt"
+	"hash"
+	"io"
+)
+
+const (
+	SHA256DigestAlgorithm = "sha256"
+	SHA512DigestAlgorithm = "sha512"
+)
```

Comment:
Not the end of the world, but there are a lot of places where these digest algorithms are hard coded strings such as Cobra commands and tests.  Worst case in the future, maybe more of those places reuse these.

Reply:
Good idea. I have an epic for post-merge polish items. I'lll add an issue for this.